packages/mui-system/src/colorManipulator.js

```diff
@@ -348,3 +348,30 @@ export function private_safeEmphasize(color, coefficient, warning) {
     return color;
   }
 }
+
+/**
+ * Blend a transparent overlay color with a background color, resulting in a single
+ * RGB color. The color space is gamma-corrected with a standard value of 2.2.
+ * @param {string} background - CSS color
+ * @param {string} overlay - CSS color
+ * @param {number} opacity - Opacity multiplier in the range 0 - 1
+ * @param {number} [gamma=1.0] - Gamma correction factor. For gamma-correct blending, 2.2 is usual.
+ */
+export function blend(background, overlay, opacity, gamma = 1.0) {
```

Comment:
I originally set the gamma correction factor to 2.2, but I had to set it back to 1.0 because that's how browsers do it and blending wouldn't match browser behavior otherwise.

It's just another case in a long serie of [Everyone Does sRGB Wrong Because Everyone Else Does sRGB Wrong
](https://erikmcclure.com/blog/everyone-does-srgb-wrong-because/).